pipelines/generic_plots.in

```diff
@@ -0,0 +1,8 @@
+imports:
+  - "$ANALYSIS_DRP_DIR/pipelines/coaddQAPlots.yaml"
+  - "$ANALYSIS_DRP_DIR/pipelines/makeQaTractTables.yaml"
+  - "$ANALYSIS_DRP_DIR/pipelines/stellarLocusPlots.yaml"
+  - "$ANALYSIS_DRP_DIR/pipelines/visitQAEllipPlots.yaml"
+  - "$ANALYSIS_DRP_DIR/pipelines/visitQAPlots.yaml"
```

Comment:
Here's a concrete proposal for how to split this up:
- `all_coadd_plots.yaml.in`: imports `coaddQAPlots.yaml` and `stellarLocuslots.yaml`
- `all_visit_plots.yaml.in`: imports `visitQAPlots.yaml` and `visitQAEllipPlots.yaml`
- `all_survey_plots.yaml.in`: empty for now.

Reply:
That looks good to me, though are @erykoff's plots in a separate pipeline as well?

Reply:
Mine can't be in a generic pipeline because they can't run generically... how that all fits in together I definitely don't understand.

Reply:
I am leaving his pipeline as a completely independent thing, that anything needs to directly refer to (especially as it seems to only be able to be included in one place right now if I understand correctly) we can modify that on future tickets.

Reply:
There is also going to be `matchVisitCatalogs.yaml`->`all_visit_plots.yaml.in` and `matchCoaddCatalogs.yaml`->`all_coadd_plots.yaml.in` as soon as Jenkins finishes for my PR.

Reply:
For @erykoff's DC2-specific pipeline, here's an extension to my proposal:
- Create a `pipelines/LSSTCam-imSim` subdirectory (matches the subdirectory in drp_pipe)
- Move `redgal_truth_plot_pipeline.yaml` there.
- Add `pipelines/LSSTCam-imSim/dc2_coadd_plots.yaml.in` which imports just that, and a builder for it.

If @taranu merges anything DC2-specific, it would also go in that subdirectory and be imported by that `dc2_coadd_plots.yaml.in`.

Reply:
or have no structure here, only pipelines, and let the appropriate camera pipeline include things in drp_pipe. I am in favor of that OR moving this machinery to drp_pipe and having it run there, but that would still leave this a big bucket of pipelines.

Reply:
i think it's better for a pipeline to self-declare where it can be used, when possible, via this kind of structure.  Having a flat directory with tons of plot pipelines, in which some are secretly but knowningly instrument-specific seems like a trap.

Reply:
Adding the instrument to the pipeline seems like it's worth doing as well, but without some mechanism for conditionally importing pipelines that looks at the instrument, I don't think it solves the problem.

Reply:
I'm not sure I follow why conditional anything is important.

Reply:
Sorry, I guess I jumped a few steps without explaining myself.  I am thinking about how to try to solve the problem of "I am a downstream pipeline author; how do I know which upstream plot-pipelines to include in my pipeline?"  Having the instrument _inside_ the pipeline but no relevant organizational structure for the pipelines themselves only seems to help with that problem _if_ there is a way they can say, "include all of the pipelines but automatically filter out the ones with the wrong instrument".